### ISSUE TITLE
Convert accesibility controls from links to buttons

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SmithKettlewellRERC/DVX-YouDescribe-issues#48]
```
Editor window has player's accessibility controls as links. They should be 
buttons.

```

Original issue reported on code.google.com by `keith...@gmail.com` on 30 Apr 2013 at 11:02
